index.js

```diff
@@ -0,0 +1,22 @@
+const menuBtn = document.querySelector('.menu-btn');
+const menu = document.querySelector('.menu');
+const menuIcon = document.querySelector('.menu-icon');
+
+function controlChecked() {
+  menu.style.maxHeight = '240px';
+  menuIcon.querySelector('.navicon').style.background = 'transparent';
+}
+
+function controlUnchecked() {
+  menuBtn.checked = false;
+  menu.style.maxHeight = null;
+  menuIcon.querySelector('.navicon').style.background = null;
+}
+
+menuBtn.addEventListener('click', () => {
+  if (this.checked) {
+    controlChecked();
+  } else {
+    controlUnchecked();
+  }
```

Comment:
Nice work with all you've done so far, but it seems a little bit that you are yet to get the full picture of the requirements.

You should please ensure that this mobile menu appears when the user clicks (or taps) the hamburger button. 😀
<img width="187" alt="right" src="https://user-images.githubusercontent.com/41163571/225261454-e26f9f05-3c9c-4c81-8f83-942a72b4b999.png">
The mobile menu is removed when the user clicks on the **X** ❌  button.
The appropriate section of the page is displayed whenever the user clicks any of the available mobile menu options, you can check [this link](https://blog.hubspot.com/marketing/jump-link-same-page) for the know-how.
The mobile menu vanishes whenever the user clicks on any of its options.

- Your functionality
![wrong](https://user-images.githubusercontent.com/41163571/225259952-9f836d57-0325-4d30-b40e-5ec4d600df78.gif)

- Required and correct functionality
![right](https://user-images.githubusercontent.com/41163571/225260264-88d1a2d8-1f1d-46c6-a796-8755121feb99.png)